Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2_test.py

```diff
@@ -965,3 +968,94 @@ def test_execute_raw_query(mocker):
     mocker.patch.object(Elasticsearch_v2.Elasticsearch, '__init__', return_value=None)
     es = Elasticsearch_v2.elasticsearch_builder({})
     assert Elasticsearch_v2.execute_raw_query(es, 'dsadf') == ES_V7_RESPONSE
+
+
+@pytest.mark.parametrize('datetime_format, expected', [
+    ('YYYY-MM-DD HH:mm:ss', 'YYYY-MM-DD HH:mm:ss'),
+    ('yyyy-MM-dd HH:mm:ss', 'YYYY-MM-DD HH:mm:ss'),
+    ('yyyy-MM-DD HH:mm:ss', 'YYYY-MM-DD HH:mm:ss'),
+    ('YYYY-MM-dd HH:mm:ss', 'YYYY-MM-DD HH:mm:ss'),
```

Comment:
You may need to import `DEFAULT_DATETIME_FORMAT`.
```suggestion
    ('YYYY-MM-DD HH:mm:ss', Elasticsearch_v2.DEFAULT_DATETIME_FORMAT),
    ('yyyy-MM-dd HH:mm:ss', Elasticsearch_v2.DEFAULT_DATETIME_FORMAT),
    ('yyyy-MM-DD HH:mm:ss', Elasticsearch_v2.DEFAULT_DATETIME_FORMAT),
    ('YYYY-MM-dd HH:mm:ss', Elasticsearch_v2.DEFAULT_DATETIME_FORMAT),
```